src/algorithms/PVT/gnuradio_blocks/rtklib_pvt_gs.h

```diff
@@ -64,7 +65,8 @@ using rtklib_pvt_gs_sptr = gnss_shared_ptr<rtklib_pvt_gs>;
 
 rtklib_pvt_gs_sptr rtklib_make_pvt_gs(uint32_t nchannels,
     const Pvt_Conf& conf_,
-    const rtk_t& rtk);
+    const rtk_t& rtk,
+    SpoofingDetector spoofing_detector);
```

Comment:
Consider that all you need to construct the detector is the config so skip this argument and just construct in the ctor.

Alternatively, follow the factory pattern in use everywhere else, and create a shared pointer that you pass in here.